projects/kalidea/kaligraphi/src/lib/molecules/kal-select/kal-select.component.ts

```diff
@@ -1,21 +1,296 @@
-import { ChangeDetectionStrategy, Component, OnInit, ViewEncapsulation } from '@angular/core';
-import { buildProviders, FormElementComponent } from '../../utils/index';
+import {
+  AfterContentInit,
+  ChangeDetectionStrategy,
+  Component,
+  ContentChildren,
+  ElementRef,
+  EventEmitter,
+  HostListener,
+  Input,
+  OnDestroy,
+  OnInit,
+  Output,
+  QueryList,
+  ViewChild,
+  ViewEncapsulation
+} from '@angular/core';
+import {Overlay, OverlayRef} from '@angular/cdk/overlay';
+import {TemplatePortal} from '@angular/cdk/portal';
+import {KalOptionComponent} from '../../atoms/kal-option/kal-option.component';
+import {filter} from 'rxjs/operators';
+import {DOWN_ARROW, ENTER, ESCAPE, SPACE, UP_ARROW} from '@angular/cdk/keycodes';
+import {ActiveDescendantKeyManager} from '@angular/cdk/a11y';
+import {FormElementComponent} from '../../utils';
 
 @Component({
   selector: 'kal-select',
   templateUrl: './kal-select.component.html',
   styleUrls: ['./kal-select.component.sass'],
   encapsulation: ViewEncapsulation.None,
-  changeDetection: ChangeDetectionStrategy.OnPush,
-  providers: buildProviders(KalSelectComponent)
+  changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class KalSelectComponent extends FormElementComponent implements OnInit {
+export class KalSelectComponent extends FormElementComponent<any> implements OnInit, OnDestroy, AfterContentInit {
 
-  constructor() {
+  @Input() disabled: boolean;
+
+  /**
+   * The placeholder displayed of the select
+   */
+  @Input() placeHolder: string;
+
+  /**
+   * Whether the component is in multiple selection mode
+   */
+  @Input() multiple: boolean;
```

Comment:
I think that you should have getter and setter to be sure that you're having a boolean value with `coerceBooleanProperty`